xchange-upbit/src/main/java/org/knowm/xchange/upbit/UpbitUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.knowm.xchange.upbit;
+
+import org.knowm.xchange.currency.CurrencyPair;
+
+public class UpbitUtils {
+
+  public static final String MARKET_NAME_SEPARATOR = "-";
+
+  private UpbitUtils() {
```

Comment:
This isn't needed

Reply:
do you mean that private constructor is not neseccery?

Reply:
Yes, it's really about enforcing so that static classes can't be sub classed, final on the class should accomplish the same thing, but neither is necessary.

Reply:
Removed.